composer.json

```diff
@@ -16,15 +16,15 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.3||~3.0",
-        "symfony/serializer": "~2.3||~3.0",
-        "symfony/console": "~2.3||~3.0",
+        "php": ">=5.3.9",
+        "symfony/framework-bundle": "~2.7||~3.0|^4.0",
```

Comment:
why `~ 2.7` and not 2.8 as the LTS?

Reply:
2.7 is LTS
ref: https://symfony.com/roadmap?version=2.7#checker